test/pure-docker/volume-config.sh

```diff
@@ -21,6 +21,6 @@ pushd ~/sourcegraph-docker
 chown -R 100:101 gitserver* lsif-server* prometheus-v2* repo-updater* searcher* sourcegraph-frontend* symbols* zoekt*
 chown -R 999:1000 redis-store-disk redis-cache-disk
 chown -R 472:472 grafana-disk
-chown -R 999:999 pgsql-disk
+chown -R 999:999 pgsql-disk codeintel-db-disk
```

Comment:
Nothing for you to do here, but FYI if you change this code in the future it is worth calling out that this means the docs need to be updated to reflect this as mentioned in the comments above.

https://github.com/sourcegraph/sourcegraph/issues/14347#issuecomment-702338141

Did you see the comments here? (wondering if there is some way we can make it stand out more or something)

Reply:
Yeah I saw, but wasn't sure what _else_ needed to be done for those migrations docs (hence the TODO you linked to).